pkg/clustermanager/cluster_manager.go

```diff
@@ -118,20 +123,22 @@ type AwsIamAuth interface {
 
 type ClusterManagerOpt func(*ClusterManager)
 
-func New(clusterClient ClusterClient, networking Networking, writer filewriter.FileWriter, diagnosticBundleFactory diagnostics.DiagnosticBundleFactory, awsIamAuth AwsIamAuth, maxWaitPerMachine time.Duration, opts ...ClusterManagerOpt) *ClusterManager {
-	retrier := retrier.NewWithMaxRetries(maxRetries, backOffPeriod)
+func New(clusterClient ClusterClient, networking Networking, writer filewriter.FileWriter, diagnosticBundleFactory diagnostics.DiagnosticBundleFactory, awsIamAuth AwsIamAuth, opts ...ClusterManagerOpt) *ClusterManager {
+	retrier := retrier.NewWithMaxRetries(maxRetries, DefaultBackOffPeriod)
```

Comment:
This is changing the behavior, right? I thought we were just adding the ability to configure some timeouts that were hardcoded before

Reply:
No I just renamed the constant on this one.